nextcloud_install_production.sh

```diff
@@ -848,7 +848,6 @@ $CHECKLIST_GUIDE" "$WT_HEIGHT" "$WT_WIDTH" 4 \
 "Contacts" "" ON \
 "IssueTemplate" "" ON \
 "PDFViewer" "" ON \
-"Extract" "" ON \
```

Comment:
It should still be default, of someone doesn't like it they can uninstall it with the menu.

Reply:
done